lib/parsers/index.ts

```diff
@@ -270,3 +270,19 @@ export function getTargetFrameworksFromProjectFile(manifestFile) {
 
   return _.uniq(targetFrameworksResult);
 }
+
+export function getTargetFrameworksFromProjectConfig(manifestFile) {
+  let targetFrameworksResult: string[] = [];
+
+  const packages = _.get(manifestFile, 'packages.package', []);
+
+  packages.forEach((item) => {
+    const targetFramework = item.$.targetFramework;
+
+    if (!_.includes(targetFrameworksResult, targetFramework)) {
```

Comment:
what happens if we have a manifest that references two different frameworks? Can we add a test for this please

Reply:
Do you have an example of a manifest that does this? I can't seem to find one.